bonnie-businessrules/src/test/java/com/cgi/bonnie/businessrules/OrderServiceTest.java

```diff
@@ -413,12 +413,12 @@ public void expectFinishOrderReturnsFalseWhenSaveFails() {
         assertFalse(orderService.finishOrder(ORDER_ID));
     }
 
-    @Test
-    public void expectGetAllOrdersCallsFindAll() {
-        orderService.getAllOrders();
-
-        verify(orderLoader).findAll();
-    }
+//    @Test
+//    public void expectGetAllOrdersCallsFindAll() {
+//        orderService.getAllOrders();
+//
+//        verify(orderLoader).findAll();
+//    }
```

Comment:
I'm not sure if we need a commented test for a deleted method, maybe we can remove this test if the tested method was also removed.